posix-configs/rpi/px4.config

```diff
@@ -7,7 +7,7 @@ df_lsm9ds1_wrapper start -R 4
 #df_mpu9250_wrapper start -R 10
 #df_hmc5883_wrapper start
 df_ms5611_wrapper start
-gps start -d /dev/pts/0
+gps start -d /dev/spidev0.0
```

Comment:
This should have a command line flag (e.g. ```-s``` for SPI) to indicate its a SPI port.